addons/website_crm_partner_assign_commission/models/sale_order.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    reseller_id = fields.Many2one('res.partner', 'Reseller', help='This field is used to track the reseller in order to generate commisions')
+    commission_rate = fields.Float('Commission Rate', default=0,
+        help="Overrides all the commissions for this order, if set to 0 the commission_rate will be used")
```

Comment:
`help="Overrides the commission rate for all product in this order. If unset or set to 0, the commission rate of each product according to the reseller's grade will be used."`